src/Main.hs

```diff
@@ -25,33 +25,36 @@ data Command
 
 main :: IO ()
 main = do
-  -- URI of the postgrest service
-  uri <- mkURI . T.pack . fromMaybe "http://localhost:3000" =<< lookupEnv "TODO_URI"
+  -- Connection to the postgrest service
+  conn <- maybe
+    (TR.TCP <$> (mkURI . T.pack . fromMaybe "http://localhost:3000" =<< lookupEnv "TODO_URI"))
+    (pure . TR.UnixSocket)
+    =<< lookupEnv "PGRST_SERVER_UNIX_SOCKET"
   -- CLI options
   opts <- execParser optsParser
   -- Run the app
-  runApp uri opts
+  runApp conn opts
```

Comment:
I think passing `uri` is fine actually, if we require (at user level) only `TODO_URI`.